docs/make.jl

```diff
@@ -0,0 +1,32 @@
+using Documenter, StatsBase
+
+makedocs(
+    format = :html,
+    sitename = "StatsBase.jl",
+    # options
+    modules = [StatsBase],
+    pages = ["index.md",
+             "weights.md",
+             "means.md",
+             "scalarstats.md",
+             "robust.md",
+             "deviation.md",
+             "cov.md",
+             "counts.md",
+             "ranking.md",
+             "sampling.md",
+             "empirical.md",
+             "signalcorr.md",
+             "misc.md",
+             "statmodels.md"]   
+)
+
+deploydocs(
+    # options
+    repo = "github.com/JuliaStats/StatsBase.jl.git",
+    target = "build",
+    julia  = "nightly",
```

Comment:
So if Base breaks something in Documenter for some time you will just go without doc updates for that time? Seems worse than the potential fear that this file will be stale (which shouldn't even change the ouput of the docs assuming the documentation still goes through).